src/expenses/controllers/expense.controller.js

```diff
@@ -0,0 +1,101 @@
+'use strict';
+
+const expenseService = require('../services/expense.service.js');
+const userService = require('../../users/services/user.service.js');
+const statusesConstants = require('../../statusesConstants.js');
+const helper = require('../../helper.js');
+
+const getAll = (req, res) => {
+  res.send(expenseService.getAll(req.query));
+};
+
+const getById = (req, res) => {
+  const id = Number(req.params.id);
+
+  const expense = expenseService.getById(id);
+
+  if (!expense) {
+    res.sendStatus(statusesConstants.NOT_FOUND);
+
+    return;
+  }
+
+  res.send(expense);
+};
+
+const create = (req, res) => {
+  if (!userService.getById(Number(req.body.userId))) {
+    res.sendStatus(statusesConstants.BAD_REQUEST);
+
+    return;
+  }
+
+  try {
+    const expense = expenseService.create(req.body);
+
+    res.statusCode = statusesConstants.CREATED;
+
+    res.send(expense);
+  } catch (err) {
+    res.sendStatus(statusesConstants.BAD_REQUEST);
+  }
+};
+
+const update = (req, res) => {
+  const id = Number(req.params.id);
+  const {
+    userId,
+    spentAt,
+    title,
+    amount,
+    category,
+  } = req.body;
+
+  const expense = expenseService.getById(id);
+
+  if (!expense) {
+    res.sendStatus(statusesConstants.NOT_FOUND);
+
+    return;
+  }
+
+  if (helper.isExpense(
+    userId,
+    spentAt,
+    title,
+    amount,
+    category)) {
```

Comment:
Better to pass object inside instead of all these values as a separate arguments